dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -779,13 +779,21 @@ public List<OrganizationUpdateTime> getUserDockstoreOrganizations(@ApiParam(hidd
     @ApiOperation(value = "See OpenApi for details")
     public List<EntryUpdateTime> getUserEntries(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User authUser,
                                                 @Parameter(name = "count", description = "Maximum number of entries to return", in = ParameterIn.QUERY) @QueryParam("count") Integer count,
-                                                @Parameter(name = "filter", description = "Filter paths with matching text", in = ParameterIn.QUERY) @QueryParam("filter") String filter) {
+                                                @Parameter(name = "filter", description = "Filter paths with matching text", in = ParameterIn.QUERY) @QueryParam("filter") String filter,
```

Comment:
not strictly part of the PR, but I wonder if we should check for a maximum count. 
That would probably be good from a safety/performance POV and alleviate the performance concern about `EntryLite`

Reply:
I have 892 entries in prod (!) (SEAB-4674](https://ucsc-cgl.atlassian.net/browse/SEAB-4674) and it executes locally in .38s with 4 JDBC statements. Given that it is already scoped to one user, and there is no pagination fallback, I think I'll spin that out as a separate ticket, unless there is an objection.

Reply:
I kinda meant something real simple, like throwing on a 1000 count limit based on your experience since we have full control over this endpoint and even more than 100 is probably useless for the UI. 

But extra ticket is ok if you insist.

Reply:
Eileen is using the total count for display in the UI. If we limit it, then we should probably add a total count http header, then the UI has to read the header. It just seems to add an extra bit of complexity at this point so close to the release.

Reply:
Logged #5085